doc/user/pim.rst

```diff
@@ -437,6 +437,14 @@ is in a vrf, enter the interface command with the vrf keyword at the end.
 
    Set the IGMP version used on this interface. The default value is 3.
 
+.. clicmd:: ip igmp max-groups (0-4294967295)
+
+   Limit the maximum amount of membership groups.
```

Comment:
I suggest this wording:

Set the maximum number of IGMP groups that the can be joined on an interface.